plugins/notifications-backend/src/database/DatabaseNotificationsStore.ts

```diff
@@ -87,24 +89,63 @@ export class DatabaseNotificationsStore implements NotificationsStore {
   };
 
   private mapToNotifications = (rows: any[]): Notification[] => {
```

Comment:
Just out of curiosity, are you intentionally not making these on the regular form

```ts
private mapToNotifications(rows: any[]): Notification[] {
```

Doing it the way you have it here, makes them instance member variables instead of a method on the contract of the type. So it's like doing an assignment in the constructor for every instantiation, leading to waste in cases where the class is created multiple times (which of course probably isn't the case here). But sometimes people do it this way to get around problems with `this` handling in callback scenarios. Should normally be used sparingly or not at all though.

Reply:
It used to be like that, I only added an extra logic there for metadata 😅 

So your suggestion is to move this method in some kind of `mapper.ts` file and have it as a function, not a member of a class?